content/rs/administering/database-operations/importing-data.md

```diff
@@ -5,17 +5,20 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-You can import data into a Redis Enterprise Software database from an
-HTTP location, FTP server, Amazon S3, or OpenStack Object Storage
-("Swift").
+You can import [backup files]({{< relref "/rs/administering/database-operations/exporting-data.md" >}})
+of a specific database to restore data.
 
-You can either import from a single file, or from multiple files in case
-you are importing from a backup of a sharded database. For additional
-details, refer to the Backup of a sharded database section in [Database
-backup]({{< relref "/rs/administering/database-operations/database-backup.md" >}}).
+You can import data from these locations:
 
-Note: Importing data via this method will erase all existing content in
-the database.
+- FTP server
```

Comment:
Please add HTTP location.
In addition- in the UI we are using "AWS S3", what are we using in the export / backup documentation?